db/seeds.rb

```diff
@@ -1,7 +1,9 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+return if User.where(name: "Test user").present?
+
+gds_organisation_id = "af07d5a5-df63-4ddc-9383-6a666845ebe9"
+
+User.create!(
+  name: "Test user",
+  permissions: %w[signin],
+  organisation_content_id: gds_organisation_id,
+)
```

Comment:
Adding the check on to the end of this expression might be neater, especially if more things are added to the db/seeds.rb file in the future. Also, maybe use `.exists?`?

So something like `) unless User.where(name: "Test user").exists?`

Reply:
Yep, that is much nice... I've amended the commit to include that change.